rust/lib/src/lib.rs

```diff
@@ -43,14 +43,19 @@ fn lock_client_return_seed(lightclient: LightClient) -> String {
 fn construct_uri_load_config(
     uri: String,
     data_dir: String,
+    chain_hint: &str,
 ) -> Result<(zingoconfig::ZingoConfig, http::Uri), String> {
     let lightwalletd_uri = construct_lightwalletd_uri(Some(uri));
 
-    let mut config = match zingolib::load_clientconfig(
-        lightwalletd_uri.clone(),
-        None,
-        zingoconfig::ChainType::Mainnet,
-    ) {
+    use zingoconfig::ChainType::*;
+    let chaintype = match chain_hint {
+        "main" => Mainnet,
+        "test" => Testnet,
+        "regression" => Regtest,
```

Comment:
those are the sames values that I can obtain from the `info` command in the chain_type?????? regression?